Sources/PaymentButtons/PaymentButtonEdges.swift

```diff
@@ -12,6 +12,9 @@ public enum PaymentButtonEdges: Int {
     /// Pill shaped corner radius.
     case rounded
 
+    /// Custom corner radius.
+    case custom(CGFloat)
+
     func cornerRadius(for view: UIView) -> CGFloat {
```

Comment:
```suggestion
    func cornerRadius(for buttonSize: CGRect) -> CGFloat {
```
🧶 : Might be overkill, but it would remove need for `import UIKit` for this enum.